### ISSUE TITLE
Bug - Accessibility - Talkback for Android does not allow user to choose a date - Calendar picker not available

### ISSUE THREAD

**Issue by maryrae-qa** [cds-snc/covid-alert-app#1587]
# Summary | Résumé

> When using an Android phone & utilizing accessibility for hearing, specifically - TalkBack, the user cannot select a date to submit as the calendar picker does not work. Talkback does not recognize the calendar.

---

> Description du bogue en un paragraphe, incluant la raison pour laquelle il s’agit d’un problème (qu’est-ce que l’utilisateur n’est pas en mesure d’accomplir?). Les captures d’écran sont très utiles.

# Steps to reproduce | Étapes à reproduire

1. Turn on Talkback for Android 
2.  Open Covid Alert App, choose menu & Enter your one-time key
3. At step1 - Enter OTK & submit 
4. At step 2 - Choose Yes & Next
5. At step 3 - Select the date & share exposures

Expected Result: User can select a date from the calendar picker & share exposures
Actual Result: TalkBack skips over the calendar picker & user **CANNOT** share exposures
**NOTE**Even if the user tries to select a date by double tapping the calendar, it does not open the picker

> Étapes consécutives (1., 2., 3., …) qui décrivent la façon de reproduire le bogue. Elles aideront les développeurs à voir ce que l’utilisateur a vu pour mieux corriger le bogue.

# Environment | Environnement logiciel

> All Android devices utilized with V6 (LG4g), V8 (Samsung S7) & V9 (Pixel2 & Motog6)
---

> Type d’appareil sur lequel le bogue est survenu. Cela comprend le modèle de l’appareil, la version du système d’exploitation (sur iOS ou Android), et toute autre précision sur l’environnement logiciel qui peut être pertinente.

# Unresolved questions / Out of scope | Questions non résolues ou hors sujet

> Are there any related issues you consider out of scope for this issue that could be addressed in the future?

---

> Y a-t-il des questions connexes que vous considérez hors sujet et qui pourraient être abordées plus tard?
